python/hsfs/engine/python.py

```diff
@@ -82,6 +83,31 @@
 except ImportError:
     pass
 
+# Decimal types are currently not supported
+PYARROW_HOPSWORKS_DTYPE_MAPPING = {
+    **dict.fromkeys(
+        [pa.uint8(), pa.uint16(), pa.int8(), pa.int16(), pa.int32()], "int"
+    ),
+    **dict.fromkeys([pa.uint32(), pa.int64()], "bigint"),
+    **dict.fromkeys([pa.float16(), pa.float32()], "float"),
+    **dict.fromkeys([pa.float64()], "double"),
+    **dict.fromkeys(
+        [pa.timestamp("ns")] + [pa.timestamp("ns", tz=tz) for tz in pytz.all_timezones],
```

Comment:
this will have to be updated after this is merged: https://github.com/logicalclocks/feature-store-api/pull/1169/files